src/python/python.cpp

```diff
@@ -94,7 +94,7 @@ struct PyGeneratorParams {
     }
   }
 
-  void SetSearchOptions(const pybind11::dict& dict) {
+  void SetSearchOptions(const pybind11::kwargs& dict) {
```

Comment:
could we list out all the valid search options here:
https://github.com/microsoft/onnxruntime-genai/blob/18adb67dce145579779dd0b6a3e2b25826a7b459/src/python/python.cpp#L182

Reply:
Do it as python help on that function? We should provide a url since the options match the config documentation.

Reply:
i mean add a comment here

Reply:
Done